Documentation/Architecture/FrameworkAndRuntime.md

```diff
@@ -0,0 +1,31 @@
+# Framework and runtime
+
+## Changes to the scene
+
+In order to use any of the MRTK functionality, you need to have a MixedRealityToolkit object in the
```

Comment:
>In order to use any of the MRTK functionality, you need to have a MixedRealityToolkit object in the scene graph, whose script handles the initialization, runtime updates, and teardown of all of the
MRTK functionality. This is the object that gets added to the scene when you use the menu option:
Mixed Reality Toolkit -> Add to Scene and Configure.

_To use the toolkit an instance of the MixedRealityToolkit script must be in your scene. To add one use the menu option: Mixed Reality Toolkit -> Add to Scene and Configure. This instance is responsible for registering, updating and tearing down services. It's also where your configuration profile is chosen._

(Scene graph is not a term people will recognize, just say scene.)